hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfo.java

```diff
@@ -134,25 +135,45 @@ public VolumeInfo build() throws IOException {
   }
 
   private long getReserved(ConfigurationSource conf) {
```

Comment:
Also, please add a basic unit test for this method.

Reply:
`TestReservedVolumeSpace` is intended to test this method specifically.